### PR TITLE
fix: grafana panels: use `mean` as the default metrics in legends 

### PR DESCRIPTION
**Pull request by fuyufjh**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

For most cases in our Grafana panels, **average** (mean) is better than **max**.

image

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

None

## Refer to a related PR or issue link (optional)

None

### CODE REVIEW

File: grafana/risingwave-dashboard.dashboard.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def timeseries_count(self,
                          title,
                          description,
                          targets,
-                         legendCols=["max"]):
+                         legendCols=["mean"]):
```
Comment (hzxa21) [inline grafana/risingwave-dashboard.dashboard.py:106]: Shall we show by setting legendCols=["max", "mean"])?

Comment (fuyufjh) [reply grafana/risingwave-dashboard.dashboard.py:106]: I considered this before, but that made the legends too wide (even wider than the graph)

image

image

Comment (fuyufjh) [reply grafana/risingwave-dashboard.dashboard.py:106]: @hzxa21 By the way, do you think max() is better for latency?

Comment (fuyufjh) [reply grafana/risingwave-dashboard.dashboard.py:106]: By the way*2, an alternative option is use `[]` (no metrics in the legend)

Comment (hzxa21) [reply grafana/risingwave-dashboard.dashboard.py:106]: > For latency, I think it's better to calculate percentiles in separate metrics, so I think using mean is fine.

+1

**Review by hzxa21 — APPROVED** [review]

**Pull request merged**